NewForce-Cohort-9/bad-blood-brotherhood
# Michael branch5

## what's new:

1.) finalized most essential html and style sheet components for cambodia while verifying other pages were functioning as intended.

## how to test:

1.) load this branch and verify function.  the most stark changes will be color choice for cambodia.
